bigflow/log.py

```diff
@@ -93,12 +99,54 @@ class LogConfigDict(TypedDict):
     log_level: typing.Union[str, int]
 
 
-def _generate_cl_log_view_link(params: dict):
-    query = quote_plus("\n".join(
-        '{}="{}"'.format(k, v)
+def _generate_cl_log_view_query(params: dict, separator: str = '='):
+    query = "".join(
+        '{}{}"{}"\n'.format(k, separator, v)
         for k, v in params.items()
-    ))
-    return f"https://console.cloud.google.com/logs/query;query={query}"
+    )
+    return query
+
+
+def get_message_for_cli(project_id, workflow_id, log_name):
+    log_name = log_name or workflow_id
+    full_log_name = f"projects/{project_id}/logs/{log_name}"
+
+    if workflow_id:
+        link = prepare_gcp_logs_link(
+            _generate_cl_log_view_query({'logName': full_log_name, 'labels.workflow_id': workflow_id}))
+        message = f'Workflow logs: {link}'
+    else:
+        link = get_bigflow_project_logs(project_id, full_log_name)
+        message = f'Bigflow project logs:{link}'
+
+    print(dedent(f"""
+           *************************LOGS LINK*************************
+           {message}
+           ***********************************************************"""))
+    _open_link_in_browser(link)
+
+
+def _open_link_in_browser(link):
+    webbrowser.open(link)
+
+
+def get_bigflow_project_logs(project_id, full_log_name):
+    pod_errors = _generate_cl_log_view_query({"severity": "WARNING"}, '>=') + _generate_cl_log_view_query(
```

Comment:
It is better to explicitly embed '>', '<=' etc symbols into field names, like
pod_errors = _generate_cl_log_view_query({"severity>=": "WARNING", "resource.type=": "k8s_pod"})
in such case `_generate_cl_log_view_query` might only join by "\n"

Reply:
Done